### PR TITLE
refactor: use autocomplete attribute on various input elements

### PR DESCRIPTION
**Pull request by Highjhacker**
## Summary

https://app.clickup.com/t/nqa57c

Should be it for MSQ.

## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

File: resources/views/profile/two-factor-authentication-form.blade.php [hunk 1]
```diff
@@ -223,7 +223,14 @@ class="justify-center w-full"
                 </div>
                 <form class="mt-8">
                     <div class="space-y-2">
-                        <x-ark-input type="password" name="password" model="confirmedPassword" :label="trans('fortify::forms.password')" />
+                        <input type="text" autocomplete="email" hidden />
```
Comment (alexbarnsley) [inline resources/views/profile/two-factor-authentication-form.blade.php]: why the hidden email field? 🤔 also `hidden` isn't actually a standard attribute for input elements is it, unless it's some custom stuff?

File: resources/views/profile/update-password-form.blade.php [hunk 2]
```diff
@@ -15,13 +15,16 @@
 
     <form class="mt-8" wire:submit.prevent="updatePassword">
         <div class="space-y-4">
+            <input type="text" autocomplete="email" hidden />
```
Comment (alexbarnsley) [inline resources/views/profile/update-password-form.blade.php]: and here 🤔

**Review by alfonsobries — COMMENTED** [review]
Other than the alex question no further comments

**Comment by Highjhacker** [comment]
Reason for the hidden inputs is for accessibility, helps with password manager apparently, I didn't knew either but it was one of the warning chrome was emitting on our forms, so handled it, see the link above for reference

https://www.chromium.org/developers/design-documents/create-amazing-password-forms

File: resources/views/profile/two-factor-authentication-form.blade.php (hunk 1 above)
Comment (ItsANameToo) [inline resources/views/profile/two-factor-authentication-form.blade.php]: @Highjhacker shouldn't this be `<input type="hidden" .... />`? assuming you meant to use a hidden input. Hidden property exists on elements but it's not how you generally make use of it for inputs

**Review by ItsANameToo — CHANGES_REQUESTED** [review]

File: resources/views/profile/two-factor-authentication-form.blade.php (hunk 1 above)
Comment (Highjhacker) [reply resources/views/profile/two-factor-authentication-form.blade.php]: Hmm unsure here 🤔 Think the examples I've seen about that thing I've described in my comment above were using it that way, but I might be wrong. Should be `type="hidden"` instead ?

Comment (ItsANameToo) [reply resources/views/profile/two-factor-authentication-form.blade.php]: show me an example 🤔  `type="hidden"` exists for a reason and I don't see why it wouldn't work for your scenario so 🤷

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**